WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -16,9 +16,14 @@ struct NewOrder: View {
         .toolbar {
             ToolbarItem(placement: .confirmationAction) {
                 Button(action: {
-                    // TODO-5405 - Add create order action
+                    viewModel.createOrder()
                 }, label: {
-                    Text(Localization.createButton)
+                    if viewModel.isLoading {
+                        ActivityIndicator(isAnimating: .constant(true), style: .medium)
```

Comment:
I think you can use `ProgressView()` here, which is the native component!

Reply:
Yep, that's what I was looking for. :) Updated to that component in d7ff8b3.